README.md

```diff
@@ -1,32 +1,31 @@
 # Bootscale
 
-Speedup applications boot by caching require calls
+Speedup applications boot by caching require calls.
+
+Spee gain depends on your number of gems. Under 100 gems you likely won't see the difference,
+but for bigger applications (300+ gems) it can make the application boot up to 30% faster.
 
-## Installation
 
-Add this line to your application's Gemfile:
+## Installation
 
 ```ruby
+# Gemfile
 gem 'bootscale', require: false
 ```
 
-And then execute:
-
-    $ bundle
-
 Then you need to add right after `require 'bundler/setup'`:
 
 ```ruby
 require 'bundler/setup'
 require 'bootscale'
-Bootscale.setup(cache_directory: '/path/to/cache/directory')
+Bootscale.setup(cache_directory: 'tmp/bootscale')
```

Comment:
Actually I don't ensure the directory exist, so it wouldn't work for copy pasters. Maybe we just default it to `tmp/`?